docs/api/api.yml

```diff
@@ -642,10 +641,6 @@ components:
       description: |
         Forbidden. Illegal state transition. (Only allowed to transition from DRAFT to SUBMITTED and from REJECTED to DRAFT.)
 
-    AlreadySubmitted:
-      description: |
-        Conflict. The deposit already exists in the submit area. Possibly due to a resubmit.
```

Comment:
Dropped in favour of `ForbiddenUpdate`?

Reply:
You like this distinction between AlreadySubmitted and ForbiddenUpdate? If we are explicit in the error message, i think they are conveying the same message

Reply:
I just was surprised as it seemed off topic. But you are right about conveying the same message. Just wanted to be sure